src/go/k8s/Makefile

```diff
@@ -78,12 +79,20 @@ docker-build: test
 docker-push:
 	docker push ${IMG}
 
+# Build the docker image
+docker-build-configurator:
+	docker build -f cli/configurator/Dockerfile -t ${CONFIGURATOR_IMG} ../
```

Comment:
I think the more common go way is to use cmd for these kind of stuff, instead of CLI

Reply:
No problem. In the next push I will rename the folder